src/pgen/radiation_advection.cpp

```diff
@@ -67,7 +64,7 @@ void ProblemGenerator(MeshBlock *pmb, ParameterInput *pin) {
   const Real vx = pin->GetOrAddReal("radiation_advection", "vx", 0.0);
   PARTHENON_REQUIRE(std::fabs(vx) < 1., "Subluminal velocity required!");
   const Real width = pin->GetOrAddReal("radiation_advection", "width", sqrt(2.0));
-  const Real kappa = pin->GetReal("opacity", "gray_kappa") * LENGTH * LENGTH / MASS;
+  const Real kappa = pin->GetReal("s_opacity", "gray_kappa") * LENGTH * LENGTH / MASS;
```

Comment:
What does `s` stand for in `s_opacity`?

Reply:
nvm figured this one out lol

Reply:
part of me wants to add an `a` to the absorption opacity class for symmetry

Reply:
hah either way